### PR TITLE
Prevent saving empty descriptions, update button size

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/17027

## Testing

**URL to test:** 

https://deploy-preview-1337--atd-moped-main.netlify.app/moped/projects/1933

**Steps to test:**

Try to save an empty description

I also updated the button sizes to all be medium (the default material ui button size)

And I fixed a bug Tilly found, https://github.com/cityofaustin/atd-moped/pull/1308#issuecomment-2087798693, you should be able to clear the public process field now. 

---
#### Ship list
- [ ] Code reviewed 
- [x] Product manager approved
- [ ] ~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable~

### CODE REVIEW

File: moped-editor/src/queries/project.js [hunk 1]
```diff
@@ -783,7 +783,7 @@ export const PROJECT_UPDATE_LEAD = gql`
 `;
 
 export const PROJECT_UPDATE_PUBLIC_PROCESS = gql`
-  mutation ProjectUpdatePublicProcess($projectId: Int!, $fieldValueId: Int!) {
+  mutation ProjectUpdatePublicProcess($projectId: Int!, $fieldValueId: Int) {
```
Comment (chiaberry) [inline moped-editor/src/queries/project.js:786]: this is a bug Tilly found while testing her PR https://github.com/cityofaustin/atd-moped/pull/1308#issuecomment-2087798693

File: moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js [hunk 2]
```diff
@@ -251,6 +251,7 @@ const ProjectsListViewTable = () => {
                 rows={data.project_list_view}
                 onSortModelChange={handleSortClick}
                 disableColumnFilter
+                localeText={{ noRowsLabel: "No projects found." }}
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js:254]: regression, we used to have this message before upgrading.

**Review by amenity — APPROVED** [review]
Works great, thank you @chiaberry! And thanks for the button size polish 💅🏻 😊

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js [hunk 3]
```diff
@@ -97,13 +97,21 @@ const ProjectSummaryProjectDescription = ({
               fullWidth
               multiline={true}
               minRows={4}
+              error={description.length < 1}
               id="moped-project-description"
               label={null}
               onChange={handleProjectDescriptionChange}
               value={description}
+              helperText={
+                description.length < 1 ? "Description cannot be blank" : ""
```
Comment (mddilley) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js:106]: 🙌

**Review by mddilley — COMMENTED** [review]
Looks so great and clean! Ready to approve, but did we want to adjust the size of the "Add subprojects" button in the project summary too?

**Comment by chiaberry** [comment]
@mddilley thanks for catching that button size, I removed the large from it

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js (hunk 3 above)
Comment (roseeichelmann) [reply moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js:106]: should users be able to type in just spaces/newlines or should we handle that as well?

**Review by mddilley — APPROVED** [review]
Yay for snackoos!! 🍫 🚀

**Review by roseeichelmann — APPROVED** [review]
This works well just left one comment about spaces but that isnt blocking this from being merged. Also sidenoteee it kinda feels off how far down the horizontal line is under the description its just all spaced out a bit weird? not related to this pr tho :p
Screenshot from 2024-05-07 15-46-12

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js (hunk 3 above)
Comment (chiaberry) [reply moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js:110]: I pushed up a change @roseeichelmann , do you think this works?

**Pull request merged**